frontend/src/metabase/search/containers/SearchApp.unit.spec.tsx

```diff
@@ -0,0 +1,212 @@
+import querystring from "querystring";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+import SearchApp from "metabase/search/containers/SearchApp";
+import { Route } from "metabase/hoc/Title";
+import {
+  setupDatabasesEndpoints,
+  setupSearchEndpoints,
+} from "__support__/server-mocks";
+import {
+  createMockDatabase,
+  createMockSearchResult,
+} from "metabase-types/api/mocks";
+import { SearchResult } from "metabase-types/api";
+
+import { SearchFilters } from "metabase/search/types";
+
+// Mock PAGE_SIZE so we don't have to generate a ton of elements for the pagination test
+jest.mock("metabase/search/containers/constants", () => ({
+  PAGE_SIZE: 3,
+}));
+
+const TRANSLATED_SIDEBAR_NAMES: Record<string, string> = {
```

Comment:
Can this be simply `SIDEBAR_NAMES`? When you prefix it with `TRANSLATED_` it feels like we're about to test translation i.e. having names in different languages.